parcel.go

```diff
@@ -35,26 +56,68 @@ func (s ParcelStore) GetByClient(client int) ([]Parcel, error) {
 
 	// заполните срез Parcel данными из таблицы
 	var res []Parcel
-
+	rows, err := s.db.Query("SELECT number, status, address, created_at FROM parcel WHERE client = :client",
+		sql.Named("client", client))
+	if err != nil {
+		return res, nil
+	}
+	defer rows.Close()
+	for rows.Next() {
+		p := Parcel{}
+		p.Client = client
+		err := rows.Scan(&p.Number, &p.Status, &p.Address, &p.CreatedAt)
+
+		if err != nil {
+			return nil, err
+		}
+
+		res = append(res, p)
+	}
 	return res, nil
 }
 
 func (s ParcelStore) SetStatus(number int, status string) error {
 	// реализуйте обновление статуса в таблице parcel
+	_, err := s.db.Exec("UPDATE parcel SET status = :new_status WHERE number = :number",
+		sql.Named("new_status", status),
+		sql.Named("number", number))
+
+	if err != nil {
+		return err
+	}
 
-	return nil
+	return err
```

Comment:
Хоть тут после проверки будет все равно `nil`, но лучше проставить явно

```suggestion
	return nil
```

Reply:
готово